frontend/src/pages/watch/sermons/index.js

```diff
@@ -0,0 +1,87 @@
+import React from "react";
+import Layout from "../../../components/layout";
+import { Breadcrumb } from "gatsby-plugin-breadcrumb";
+import {
+  BookIcon,
+  CalendarIcon,
+  SeriesIcon,
+  UserIcon,
+} from "../../../components/svgs";
+import { StaticImage } from "gatsby-plugin-image";
+
+const SermonPage = ({ pageContext }) => {
+  const {
+    breadcrumb: { crumbs },
+  } = pageContext;
+  return (
+    <Layout>
+      <div className="px-4 w-full flex items-center flex-col">
+        <div className="max-w-container w-full flex flex-col py-[1.375rem] lg:pt-10 lg:pb-[22.8125rem] gap-y-[2.1875rem] lg:gap-y-[3.75rem] items-center">
+          <div className="w-full pl-[2px] lg:pl-0">
+            <Breadcrumb crumbs={crumbs} crumbSeparator=" > " />
+          </div>
+          <div className="flex flex-col gap-y-[0.75rem] lg:gap-y-[3.75rem] items-center max-w-[61.25rem] w-full pb-[3.98625rem] lg:pb-0">
+            <h1 className="text-2xl lg:text-4xl leading-tighter font-semibold lg:font-bold mb-0">
+              Psalm 133
+            </h1>
+            <div className="flex flex-col lg:flex-row lg:gap-x-10 items-center pt-[0.9375rem] lg:pt-0 gap-y-5 lg:gap-y-0">
+              <div className="flex flex-col gap-y-1 lg:gap-y-10 text-base lg:text-xl font-medium w-full lg:w-auto">
+                <div className="flex gap-x-2">
+                  <CalendarIcon />
+                  <div>August 13, 2023</div>
+                </div>
+                <div className="flex flex-col gap-y-1 lg:gap-y-3">
+                  <div className="flex gap-x-2">
+                    <UserIcon />
+                    <div>Pastor Pete Dahlem</div>
+                  </div>
+                  <div className="flex gap-x-2">
+                    <BookIcon />
+                    <div>Psalm 133</div>
+                  </div>
+                  <div className="flex gap-x-2">
+                    <SeriesIcon />
+                    <div>Selah</div>
+                  </div>
+                </div>
+              </div>
+              <div className="w-full max-w-[45rem]">
+                <div className="relative">
+                  <StaticImage
+                    alt="Psalm 133"
+                    src="../../../images/Sermon-Ad-Selah-April.png"
+                    className="max-h-[25rem]"
```

Comment:
you mean this? i have set max height.

Reply:
Is the size of the video player static, or dependent on the size of the sermon series picture? I'm just worried about the backend eventually having different sermon series pictures of various size ratios

Reply:
The only issue will be size of the image is smaller than 400 px, which if we set exact size for anything smaller than 400px for desktop, it will be pixelated. Unless redize the container based upon the size of the image we can get from backend.  What do you want to do?

Reply:
Let's assume that nothing will be resized in the backend - all resizing done in frontend. Given that these pictures will be from ads team, let's also assume that everything is >400px. 

My problem is, let's say right now we have a 1080x1920px picture, and then later on we receive a 1080x1680 picture. How will we render the video player in each of these scenarios? Will the pictures be cropped? Will there be extra white/black spaces on certain edges? Or will the size of the video player be dynamically sized to the exact size of the picture? 

You only specified the max-height == 25rem. Does that mean that the width will be variable, depending on the height/width ratio of the picture?

Reply:
yes.  i can do max-width only. as long as we have one of the set up for max than it will resize the picture that matches the max height/ width so here is the scenario: max height = 25rem / 400px  which means 
1080 w x 1920 h = 225 w x 400 h or 1080 w x 1680 h = 257 w x 400 h  for see this case i think i should do max width so the width should be same and the height will vary.  I do this way so the image would not be too smoosh together.